Lombiq.Tests.UI/Helpers/ReliabilityHelper.cs

```diff
@@ -99,7 +101,7 @@ public static class ReliabilityHelper
         {
             throw new TimeoutException(
                 "The process didn't succeed with retries before timing out " +
-                $"(timeout: {result.Wait.Timeout}, polling interval: {result.Wait.PollingInterval}).");
+                $"(timeout: {result.Wait.Timeout:c}, polling interval: {result.Wait.PollingInterval.ToTechnicalString()}).");
```

Comment:
We now have three different ways of displaying a `TimeSpan` in one file - that seems like overkill:

![image](https://github.com/Lombiq/UI-Testing-Toolbox/assets/164360/026a9d3d-46ae-4bd4-af43-46746804d1d5)

You don't need the `c` format specifier here, because it seems to be the default anyway:

![image](https://github.com/Lombiq/UI-Testing-Toolbox/assets/164360/4eb1573e-a54b-49aa-bae3-a8f3773e65a4)

I suggest to use the same approach here as in lines 69-71, and remove the new extension method on `TimeSpan`.

Reply:
Yep, `StringHelper` should be used as well. I've used `c` before adding `ToTechnicalString()` and apparently this one usage I forgot to change.

Otherwise, `ToTechnicalString()` is necessary, since not all `ToString()`s of `TimeSpan` are in formatted strings.

Reply:
Currently, `TimeSpanExtensions.ToTechnicalString()` is not used anywhere in OSOCE, so I feel an urge to have it removed. `{timeSpan:c}` will already produce a culture-invariant string representation, and judging from the lack of analyzer warnings on the `timeSpan.ToString("c");` method body, that shouldn't pose a problem.

In fact, `timeSpan.ToString();` will produce the same result.